### ISSUE TITLE
Two webviews in one winform won't work 

### ISSUE THREAD

**Issue by PaulYBChiang** [MicrosoftEdge/WebView2Feedback#204]
It's really weird that if I add two webview2s in one WinForm, only one works and the other always sleeping. The code is pretty straight forward.

       private void Form1_Load(object sender, EventArgs e)
        {
            webView21.Source = new Uri(url1);
            webView22.Source = new Uri(url2);
        }

[AB#26584427](https://microsoft.visualstudio.com/90b2a23c-cab8-4e7c-90e7-a977f32c1f5d/_workitems/edit/26584427)

**Comment by nicholasdgoodman** [comment]
I have observed something similar yesterday - it appears timing related. I could get multiple views on one form only if I put a delay between their creation.

**Comment by david-risney** [comment]
Thanks for reporting the issue! We'll look into this a get back to you.

**Comment by mikemeinz** [comment]
@PaulYBChiang
@nicholasdgoodman

This works for me.

```
using System;
using Imports Microsoft.Web.WebView2.Core;
public partial class Main
{
    public Main()
    {
        this.Load += WebView2Test_Load;
    }

    private string HTMLText1 = "<!doctype html><html lang=\"en\"><head><title>Test 1</title></head><body><p>Test 1</p></body></html>";
    private string HTMLText2 = "<!doctype html><html lang=\"en\"><head><title>Test 2</title></head><body><p>Test 2</p></body></html>";

    public void WebView2Test_Load(object sender, EventArgs e)
    {
        InitializeAsync();
    }

    public async void InitializeAsync()
    {
        await WebView21.EnsureCoreWebView2Async();
        WebView21.NavigateToString(HTMLText1);
        await WebView22.EnsureCoreWebView2Async();
        WebView22.NavigateToString(HTMLText2);
    }
}
```

**Comment by oggy22** [comment]
We are working on the issue. The problem seems to be the same `UserDataFolder`. Even though officially supported it seems to be problematic in Winforms, so a temporary workaround for this issue before it's fixed would be choosing not default `UserDataFolder` for the second webview2.

        var createProps = new CoreWebView2CreationProperties();
        createProps.UserDataFolder += "2";
        webView22.CreationProperties = createProps;

**Comment by mikemeinz** [comment]
@oggy22 

**CoreWebView2CreationProperties()** is in **Microsoft.Web.WebView2.Wpf.**  It doesn't seem right to use a class that is part of the WPF part of WebView2 in a WinForms app. Furthermore, **CreationProperties()** is not in WebView2.

WebView2 Version 0.9.628-prerelease

```
Imports Microsoft.Web.WebView2.Core
Imports Microsoft.Web.WebView2.Wpf


 24       Dim createProps As New CoreWebView2CreationProperties
 25       createProps.UserDataFolder += "2"
 26       WebView22.CreationProperties = createProps
 27       Await WebView22.EnsureCoreWebView2Async()
```

VB .NET IDE gives these errors: for lines 25 and 26
```
Error	BC30652	Reference required to assembly 'WindowsBase, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' containing the type 'DependencyObject'. Add one to your project.	WebMon	C:\Users\Me\Documents\Visual Studio 2017\Projects\WebCam\WebMon\Main.vb	25	Active
Error	BC30456	'CreationProperties' is not a member of 'WebView2'.	WebMon	C:\Users\Me\Documents\Visual Studio 2017\Projects\WebCam\WebMon\Main.vb	26	Active
```

**Comment by champnic** [comment]
This should be fixed in [WebView2 SDK 1.0.674-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.674-prerelease). If this is still failing please let us know. Thanks!
